frontend/src/Comment.js

```diff
@@ -0,0 +1,35 @@
+import './comment.css';
+
+function Comment({ children }) {
+  const regExp = /(\d{2}:\d{2}:\d{2}\.\d{3}\s*-->\s*\d{2}:\d{2}:\d{2}\.\d{3})/;
```

Comment:
You may change the constant name in order to tell that this is a time code format.

Reply:
Done